integration_test/metadata/integration_metadata.go

```diff
@@ -87,11 +87,15 @@ type LogField struct {
 	// A list of platforms that this field is not available on.
 	// Examples: debian-11.
 	UnavailableOn []string `yaml:"unavailable_on,omitempty"`
+	// Annotations/footnotes about the field.
+	Notes []string `yaml:"notes,omitempty" validate:"omitempty,unique"`
```

Comment:
Does this need to be `[]*string` too, for anchors to work?

Reply:
Not really, it would work either way. The only thing using pointers would buy you is that YAML aliases would be aliased in Go, rather than copied. Since these are strings, I didn't bother. None of the notes use pointers.